### PR TITLE
RWA-275 wa system user

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-275

### CODE REVIEW

**Review by indvamshi — DISMISSED** [review]

**Comment by iguannago** [comment]
These 4 FTs fail because of a dependency with the CCD def PR: https://github.com/hmcts/ia-ccd-definitions/pull/396

**Review by lewis150193 — DISMISSED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -8,13 +8,17 @@ def component = "task-configuration-api"
 
 def secrets = [
   'wa-${env}' : [
-      secret('wa-test-law-firm-a-username', 'TEST_WA_LAW_FIRM_USERNAME'),
-      secret('wa-test-law-firm-a-password', 'TEST_WA_LAW_FIRM_PASSWORD'),
-      secret('wa-idam-client-id', 'WA_IDAM_CLIENT_ID'),
-      secret('wa-idam-client-secret', 'WA_IDAM_CLIENT_SECRET'),
+    secret('wa-test-law-firm-a-username', 'TEST_WA_LAW_FIRM_USERNAME'),
+    secret('wa-test-law-firm-a-password', 'TEST_WA_LAW_FIRM_PASSWORD'),
+
```
Comment (alexbaeza) [inline Jenkinsfile_CNP:13]: we don't need this one anymore

Comment (iguannago) [reply Jenkinsfile_CNP:13]: We do, this is used in the FT to create CCD cases. Ravi and Marcin agreed to do it this way.

**Review by alexbaeza — APPROVED** [review]

**Comment by alexbaeza** [comment]
Looks good to me We will need to create the users and add them to the vault in other environments

**Pull request merged**